config/functions

```diff
@@ -136,7 +136,7 @@ safe_remove() {
 
   [ -z "${path}" ] && return 0
 
-  if [ -f "${path}" -o -d "${path}" ]; then
+  if [ -f "${path}" -o -d "${path}" -o -L "${path}" ]; then
```

Comment:
Why not just using **-e** ?

Reply:
`-e` will return `false` for for a symbolic link that references a non-existent target meaning it would not be possible to delete a "broken" symbolic link.
```
if [ -e "${path}" -o -L "${path}" ]; then
```
would work fine.